osu.Game/Online/API/Requests/SearchBeatmapSetsRequest.cs

```diff
@@ -50,7 +52,8 @@ public class SearchBeatmapSetsRequest : APIRequest<SearchBeatmapSetsResponse>
             SearchLanguage language = SearchLanguage.Any,
             IReadOnlyCollection<SearchExtra> extra = null,
             IReadOnlyCollection<ScoreRank> ranks = null,
-            SearchPlayed played = SearchPlayed.Any)
+            SearchPlayed played = SearchPlayed.Any,
+            SearchExplicit explicitMaps = SearchExplicit.Hide)
```

Comment:
another example of a random mutation of the variable `explicitMaps`. It's like you came up with a new variation for each usage, even when the usages are denoting the same thing.